ui/pages/onboarding-flow/recovery-phrase/confirm-recovery-phrase.js

```diff
@@ -76,21 +75,21 @@ export default function ConfirmRecoveryPhrase({ secretRecoveryPhrase = '' }) {
         textAlign={TEXT_ALIGN.CENTER}
         marginBottom={4}
       >
-        <Typography
-          variant={TypographyVariant.H2}
-          fontWeight={FONT_WEIGHT.BOLD}
+        <Text
+          variant={TextVariant.headingLg}
+          fontWeight={TextVariant.bodyMdBold}
```

Comment:
Incorrectly using TextVariant for the `fontWeight` prop here. It should use FontWeight.Bold or Devin should be abled to discern that `headingLg` is bold already.
```suggestion
          fontWeight={FontWeight.Bold}
```